app/controllers/users_controller.rb

```diff
@@ -34,26 +34,23 @@ def update
   end
 
   def show
-    if current_user
       @user = User.find(params[:id])
-    else 
-      render 'home#index'
-    end
   end
 
   def destroy
     @user = User.find(params[:id])
-      if @user.destroy
-        flash[:notice] = 'User has been destroyed'
-        redirect_to users_path
-      else
-        flash[:alert] = 'Something went wrong'
-        redirect_to users_path
-      end
+    if @user.destroy
+    flash[:notice] = 'User has been destroyed'
+    redirect_to users_path
+    else
+    flash[:alert] = 'Something went wrong'
+    redirect_to users_path
```

Comment:
teraz coś w ogóle się z wcięciami pokićkało 

i redirect dalej w ifie

wcięcia powinny wyglądać tak

class Klasa
  def metod
    if a == b
      return
    end
  end
end

Przy każdym zagnieżdżeniu jest wcięcie, jeśli coś się w czymś nie zagnieżdża to nie robimy wcięcia.